Podfile

```diff
@@ -15,6 +15,7 @@ target 'Lets Do This' do
     pod 'GoogleAnalytics', '3.13.0'
     pod 'NewRelicAgent', '5.3.4'
     pod 'NSString+RemoveEmoji', '0.1.0'
+    pod 'TapjoySDK'
```

Comment:
This link to [Getting Started for advertisers](https://home.tapjoy.com/tech/product-overview/integration-advertisers/integrating-the-advertiser-sdk) seems to have the methods v10 is expecting: 
````
[Tapjoy requestTapjoyConnect:@"" secretKey:@"" options:@{ TJC_OPTION_ENABLE_LOGGING : @(YES) } ];
````